README.md

```diff
@@ -29,8 +29,10 @@
 
 Make sure you have Go 1.10+ (required for [go-releaser](https://goreleaser.com/))
 
-- Ubuntu: https://launchpad.net/~longsleep/+archive/ubuntu/golang-backports or https://github.com/golang/go/wiki/Ubuntu
-- OSX: `brew install go`
+- Ubuntu: https://launchpad.net/~longsleep/+archive/ubuntu/golang-backports or https://github.com/golang/go/wiki/Ubuntu [Easier but maybe older] **(recommended)**
+- Linux: [GoLang official install guide](https://golang.org/doc/install#tarball) [Harder but up to date] 
+- OSX: [GoLang official install guide](https://golang.org/doc/install#macos)
+- Windows: Not supported, there will be ways to run it on Windows if you use Docker.
```

Comment:
I tested the Windows build and it works fine. Did you find out differently?

Reply:
Is there documentation on how to compile it in go as I think this section is to explain how to do that.  This section didn't have the generic Linux section before so I added that.  Then I added the windows section while making assumptions that there wasn't a Windows build as it wasn't documented prior.  I mostly wanted to add official go docs for installing go not just what we use.